### PR TITLE
#1 step by step migrations intro

### PR DESCRIPTION
**Pull request by naazy**
#1 step by step migrations intro

@nelsonic ready to be reviewed :pray: 📿

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+config.env
```
Comment (nelsonic) [inline .gitignore]: @naazy we need to move _away_ from `config.env` towards simply `.env` 😉

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+Running migrations on node postgres
+
+### Step 1
+Add the sql migrations you would like to run to the migrations folder (1 migration per file).
+**The file names should start with the date in the format yy-mm-dd** (this is to ensure the migrations are run in the correct order - from top to bottom)
+
+### Step 2
+Add the environment variables:
+- DATABASE_URL
+- DATABASE_NAME
```
Comment (nelsonic) [inline README.md]: technically `DATABASE_URL` _includes_ the `DATABASE_NAME` ...

**Review by nelsonic — COMMENTED** [review]
@naazy script looks great. instructions are clear. 🎉 
`DATABASE_NAME` is mentioned in README but not in the `code`. can we _remove_ it?

File: README.md (hunk 2 above)
Comment (naazy) [reply README.md]: Good point

**Comment by naazy** [comment]
Thanks @nelsonic - will remove DATABASE_NAME 😄

**Comment by naazy** [comment]
Comments addressed @nelsonic 🌻

**Review by nelsonic — APPROVED** [review]
@naazy looks good. thanks! 🎉

**Pull request merged**